packages/carbon-graphs/src/js/helpers/axis.js

```diff
@@ -893,7 +893,16 @@ const createAxes = (axis, scale, config, canvasSVG) => {
       )})`,
     )
     .call(axis.x)
-    .call(resetD3FontSize);
+    .call(resetD3FontSize)
+    .attr('id', 'x')
+    .selectAll('text')
+    .style('text-anchor', () => {
+      if (config.axis.x.ticks.rotateAngle !== undefined) {
```

Comment:
`rotateAngle` is grammatically incorrect and also unclear without reading the documentation. Can we rename this to `tickLabelsRotation`?

```suggestion
      if (config.axis.x.ticks.tickLabelsRotation !== undefined) {
```